app/utils/user/index.ts

```diff
@@ -91,6 +91,10 @@ export function isSystemAdmin(roles: string): boolean {
     return isRoleInRoles(roles, Permissions.SYSTEM_ADMIN_ROLE);
 }
 
+export function isChannelAdmin(roles: string): boolean {
+    return isRoleInRoles(roles, Permissions.CHANNEL_ADMIN_ROLE);
+}
```

Comment:
do we really need this?

Reply:
It's up to you -- I thought it fit, since we already have `isGuest` and `isSystemAdmin` just above.